extensions/amp-video/0.1/flexible-bitrate.js

```diff
@@ -293,3 +299,20 @@ function currentSource(video) {
     })
   );
 }
+
+/**
+ * @private
+ * @param {!Element} videoEl
+ * @return {number}
```

Comment:
Please specify that it returns `[0, 1]` because with the `percentage` keyword it could also be `[0, 100]`